### PR TITLE
DDF-2259 Fixed registry itest date issue

### PR DESCRIPTION
**Pull request by clockard**
#### What does this PR do?
Fixes registry itest bug
#### Who is reviewing it (please choose AT LEAST two reviewers that need to approve the PR before it can get merged; if a component team is listed, at least one of its members needs to approve)?
@mcalcote @ani6gup 
#### Choose 2 committers to review/merge the PR (please choose ONLY two committers from below, delete the rest).
@coyotesqrl
@pklinef

#### How should this be tested?
Verify build passes
#### What are the relevant tickets?
DDF-2259
#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

### CODE REVIEW

**Comment by mcalcote** [comment]
Assuming bamboo passes :+1:

File: distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestRegistry.java [hunk 1]
```diff
@@ -220,7 +220,7 @@ public void testCswRegistryStoreUpdate() throws Exception {
                         FederationAdminService.class);
                 federationAdminServiceImpl.updateRegistryEntry(Library.getRegistryNode(regID,
                         "New Node Name",
-                        "2015-02-26T17:16:34.996Z"), destinations);
+                        "2016-02-26T17:16:34.996Z"), destinations);
```
Comment (coyotesqrl) [inline distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestRegistry.java]: If 2015 failed, why does 2016 pass? Does it need to be within one year from test run? Is it something else fragile that will cause this test to fail next year? Should this date string be generated dynamically so it is future-proof?

Comment (clockard) [inline distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestRegistry.java]: The earlier fix caused part of the code to run that wasn't before. This code rejects updates if the modified time of the update is older than the currently existing modified time. There is a 2015 modified date string in the test data. This date just needs to be sometime after that date so i changed it to 2016.

**Comment by coyotesqrl** [comment]
LGTM. 👍

File: distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestRegistry.java (hunk 1 above)
Comment (stustison) [inline distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestRegistry.java]: For the future, when it comes to dates in tests, it is always best to generate dates yourself, lest you run into issues down the road. See the security itests for examples.

**Pull request merged**